shell/edit/auth/ldap/config.vue

```diff
@@ -95,305 +95,303 @@ export default {
 
 <template>
   <div @update:value="$emit('update:value', model)">
-    <template>
```

Comment:
not sure if it's related to this change, but can't validate this change due to it, when i go to any of the ldap auth providers the page fails to render. console error

![image](https://github.com/user-attachments/assets/ce809657-fa3e-4b57-b9f1-9464749e4edd)

Reply:
Thanks for the heads up. I can investigate and create another issue if one is required.

Reply:
We get this error in `master` without this change present and I've traced the issue down to a circular reference in our components.

We import `saml.vue` in `config.vue`:

https://github.com/rancher/dashboard/blob/14089aba62240aa4f7af9eb27511e343e2f352e0/shell/edit/auth/ldap/config.vue#L8

And in `saml.vue`, we import `config.vue`:

https://github.com/rancher/dashboard/blob/14089aba62240aa4f7af9eb27511e343e2f352e0/shell/edit/auth/saml.vue#L12

I think that fixing this issue will be outside of the scope of this PR, so I'm working on #11934 to track the issue.